docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -32,5 +63,49 @@ Permissions at cluster scope.
 * get pods and pod logs
 * view deployments
 * get deployments
+* view routes
+* view egress firewalls
+* view roles and rolebindings
+* patch persistentvolumeclaims
+* view secretstores
+
+## backplane-acs-ingress: `openshift-ingress`
+
+- ACS team needs get/list/watch access to inspect the routers and logs
+
+* view deployments
+* view pods
+* view pod logs
+
+## backplane-acs-monitoring: `openshift-monitoring`
+
+- ACS team needs get/list/watch access to the openshift monitoring prometheus-related pods, statefulsets and services
+
+* view pods
+* view pods/log
+* view statefulsets
+* view services
+
+## backplane-acs-rhacs-observability: `rhacs-observability`
+
+- ACS team needs to inspect the observability resources
+
+* view alertmanagerconfigs
+* view alertmanagers
+* view podmonitors
+* view probes
+* view prometheuses
+* view prometheusrules
+* view servicemonitors
+* view thanosrulers
+* view observabilities
+* view grafanas
+* view grafanadatasources
+* view grafanadashboards
+
+- ACS team needs to be able to increase the prometheus storage and restart prometheus
+
+* patch persistentvolumeclaims
+* patch statefulsets
```

Comment:
Missing get/list/watch for pvc and statefulset in rhacs-observability?

Reply:
get/list/watch is allowed in `rhacs-*`